modules/rest-http/src/main/java/org/apache/ignite/internal/processors/rest/protocols/http/jetty/GridJettyObjectMapper.java

```diff
@@ -264,4 +292,36 @@ private void writeException(Throwable e, JsonGenerator gen) throws IOException {
             }
         }
     };
+
+    /** Custom serializer for {@link java.sql.Timestamp}. */
+    private static final JsonSerializer<Timestamp> IGNITE_TIMESTAMP_SERIALIZER = new JsonSerializer<Timestamp>() {
+        /** {@inheritDoc} */
```

Comment:
redundant javadoc

Reply:
Thanks, 
I missed javadocs in similar places, actually.